### ISSUE TITLE
Does it work with Typescript in Vue.js files?

### ISSUE THREAD

**Issue by sebastianhaberey** [1nVitr0/plugin-vscode-invert-if#35]
Hi,

i'm trying to use this extension in the `<script lang="ts">` - block of a .vue file, but I don't get the refactoring options:

![issue](https://github.com/user-attachments/assets/dc7c25b1-f650-41aa-81d1-7b885f9afd05)

Is this because the code is in a .vue file?

**Comment by 1nVitr0** [comment]
Yes, currently embedded languages are not supported. I haven't looked into how they can be, when I do I'll get back to you. Might take me a few weeks though.

**Comment by 1nVitr0** [comment]
@sebastianhaberey I will temporarily  add vue in the inbuilt js / ts  support, though for the long run, I want to support all embedded languages. I should be able to finish the temporary fix over the holidays.

**Comment by 1nVitr0** [comment]
@sebastianhaberey I have just released the support for embedded languages, and a plugin for vue support. There seem to be some issues with indentation, I'll have a look at those.

For now, when the [vue support extension](https://marketplace.visualstudio.com/items?itemName=1nVitr0.invert-if-vue) is installed, all commands should work inside vue files for both typescript and javascript. Let me know if you have any issues with the extensions.

**Comment by sebastianhaberey** [comment]
Hi I just wanted to say this is great, and I think the decision to support all embedded languages seems like a consistent decision. I cannot try it out right now because I'm on vacation but I'll be sure to report back once I'm home. Thanks for this!
